CODE/Sources/PennyBOT/Constants.swift

```diff
@@ -3,8 +3,10 @@ import DiscordBM
 
 enum Constants {
     static let internalChannelId = "441327731486097429"
+    static let botDevUserId = "290483761559240704"
     static let botToken: String! = ProcessInfo.processInfo.environment["BOT_TOKEN"]
     static var botId: String! = ProcessInfo.processInfo.environment["BOT_APP_ID"]
+    static var loggingWebhookUrl: String! = ProcessInfo.processInfo.environment["LOGGING_WEBHOOK_URL"]
```

Comment:
Need to add `LOGGING_WEBHOOK_URL` to aws.
First need to create a webhook in Discord, of course.